api/chat_main_handler.go

```diff
@@ -245,7 +224,7 @@ func (h *ChatHandler) OpenAIChatCompletionAPIWithStreamHandler(w http.ResponseWr
 		log.Printf("%+v\n", chatPrompt)
 	}
 
-	msgs, err := getAskMessages(h, ctx, w, chatSession, chatUuid, false)
+	msgs, err := h.chatService.getAskMessages(chatSession, chatUuid, false)
 	if err != nil {
 		RespondWithError(w, http.StatusInternalServerError, fmt.Errorf("fail to collect messages: %w", err).Error(), err)
 		return
```

Comment:
:robot: OpenAI

This patch changes the call to `getAskMessages` to use the `chatService` instance instead of calling it as a standalone function.

This change is an improvement because it makes the code more modular and easier to test.

Please confirm that the `chatService` instance is initialized correctly and implements the `getAskMessages` function before merging this patch.

Also, please update the comments on the pull request to reflect this change.

<!-- This is an auto-generated comment by OpenAI -->